utils/internal/custom_markdown_documenter.ts

```diff
@@ -0,0 +1,1430 @@
+// Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the
+// MIT license. See LICENSE in the project root for license information.
+
+import {
```

Comment:
so there is no way to customize the site without brining in all of this code?

Reply:
Where is it taken from and what are the changes? Can we upstream the changes?

Reply:
From `https://github.com/microsoft/rushstack/blob/main/apps/api-documenter/src/documenters/MarkdownDocumenter.ts`. This only really suits us because we augment the generator just for Docusaurus.